src/midend/frontier_reuse_analysis.cpp

```diff
@@ -0,0 +1,91 @@
+#include <graphit/midend/frontier_reuse_analysis.h>
+
+namespace graphit {
+void FrontierReuseAnalysis::analyze(void) {
+	for (auto func: mir_context_->getFunctionList()) {
+		ReuseFindingVisitor visitor(mir_context_);
+		func->accept(&visitor);	
+	}
+}
+bool FrontierReuseAnalysis::ReuseFindingVisitor::is_frontier_reusable(mir::StmtBlock::Ptr stmt_block, int index, std::string frontier_name) {
```

Comment:
Yeah this is a merge from Ajay's pull request. I can remove it but I thought we are gonna merge this anyways so that GPU merge will be slightly easier later on?

Reply:
If I understand correctly, we are not using this anymore. So we shouldn't include it in this pull request. It will get refactored and merged with the GPU backend branch anyway. As it is this PR has a lot of features. If you can drop this pass and make sure all the test cases are running, we should drop it. There is another file above related to this.